src/Network/Algorand/Def.hs

```diff
@@ -0,0 +1,105 @@
+-- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+-- | Definitions to work with Algo, grouped in this module for convenience.
+module Network.Algorand.Def
```

Comment:
I somewhat dislike `Def` as the module name because I associate `Def` with `Default` rather than `Definition`. If you want this module to stand for "Definitions", maybe let's call it `Network.Algorand.Definitions`? The length in this case looks acceptable to me, it's not too long because it's only at the third level.

Reply:
okay, I do not mind